### PR TITLE
[microTVM][CI] Rename ci_qemu to ci_cortexm

### PR DESCRIPTION
**Pull request by mehrdadh**
This PR is the first step of this tracking issue:
https://github.com/apache/tvm/issues/12280

cc @Mousius @alanmacd @areusch @driazati @gromero

### CODE REVIEW

File: ci/jenkins/Jenkinsfile.j2 [hunk 1]
```diff
@@ -56,7 +56,7 @@ ci_gpu = 'tlcpack/ci-gpu:20220801-060139-d332eb374'
 ci_cpu = 'tlcpack/ci-cpu:20220715-060127-37f9d3c49'
 ci_wasm = 'tlcpack/ci-wasm:20220715-060127-37f9d3c49'
 ci_i386 = 'tlcpack/ci-i386:20220715-060127-37f9d3c49'
-ci_qemu = 'tlcpack/ci-qemu:20220630-060117-558ba99c7'
```
Comment (driazati) [inline ci/jenkins/Jenkinsfile.j2:59]: This is referenced in a few other places (`Test.groovy.j2`, `Build.groovy.j2`) that you also might have to change

**Review by driazati — COMMENTED** [review]
I pushed to this branch https://github.com/apache/tvm/tree/micro/rename_qemu so https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/micro%2Frename_qemu/1/pipeline will test the new Jenkinsfile (so if you have any updates make sure to push them to both the apache/tvm repo and your fork)

File: ci/jenkins/Jenkinsfile.j2 (hunk 1 above)
Comment (mehrdadh) [reply ci/jenkins/Jenkinsfile.j2:59]: fixed these.

**Comment by mehrdadh** [comment]
@driazati there was an issue with docker naming with branch `micro/rename_qemu`. So I pushed to a different branch `micro_rename_qemu`:
https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/micro_rename_qemu/2/pipeline/

**Comment by mehrdadh** [comment]
@driazati CI seems green here:
https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/micro_rename_qemu/6/pipeline

**Review by driazati — APPROVED** [review]

**Pull request merged**